conftest.py

```diff
@@ -14,6 +14,25 @@ def pytest_addoption(parser):
     parser.getgroup('cfme', 'cfme: options related to cfme/miq appliances')
 
 
+def pytest_plugin_registered(plugin, manager):
```

Comment:
this function looks like a leftover from the first conftest integration, why was it added again?